src/app/src/components/DashboardMergeFacilityControls.jsx

```diff
@@ -0,0 +1,140 @@
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import { toast } from 'react-toastify';
+import get from 'lodash/get';
+
+import { facilityDetailsPropType } from '../util/propTypes';
+
+const mergeControlsStyles = Object.freeze({
+    containerStyles: Object.freeze({
+        width: '100%',
+        padding: '10px 0',
+        margin: '10px 20px',
+        display: 'flex',
+        justifyContent: 'start',
+    }),
+    buttonStyles: Object.freeze({
+        margin: '0 20px 10px 0',
+    }),
+    labelStyles: Object.freeze({
+        fontSize: '16px',
+    }),
+});
+
+export default function DashboardMergeFacilityControls({
+    handleMerge,
+    handleFlip,
+    merging,
+    error,
+    disabled,
+    toMergeData,
+    targetData,
+}) {
+    const [dialogIsOpen, setDialogIsOpen] = useState(false);
+    const [mergingFacilities, setMergingFacilities] = useState(false);
+
+    const mergeFacilities = () => {
+        setMergingFacilities(true);
+
+        return handleMerge();
+    };
+
+    useEffect(() => {
+        if (!merging && mergingFacilities) {
+            setDialogIsOpen(false);
+            setMergingFacilities(false);
+            toast('Facilities were merged');
+        }
+    }, [merging, mergingFacilities, setDialogIsOpen, setMergingFacilities]);
+
+    const targetName = get(targetData, 'properties.name', '');
+    const targetID = get(targetData, 'id', '');
+    const toMergeName = get(toMergeData, 'properties.name', '');
+    const toMergeID = get(toMergeData, 'id', '');
+
+    return (
+        <div style={mergeControlsStyles.containerStyles}>
+            <Button
+                onClick={() => setDialogIsOpen(true)}
+                disabled={merging || disabled}
+                variant="contained"
+                color="secondary"
+                style={mergeControlsStyles.buttonStyles}
+            >
+                Merge facilities
+            </Button>
+            <Button
+                onClick={handleFlip}
+                disabled={merging || disabled}
+                variant="contained"
+                color="primary"
+                style={mergeControlsStyles.buttonStyles}
+            >
+                Flip facilities
+            </Button>
+            {merging && <CircularProgress />}
+            {error && (
+                <span style={{ color: 'red' }}>
+                    An error prevented merging those facilities
+                </span>
+            )}
+            <Dialog open={dialogIsOpen}>
+                {dialogIsOpen ? (
+                    <>
+                        <DialogTitle>
+                            Merge {get(toMergeData, 'properties.name', '')} into{' '}
+                            {get(targetData, 'properties.name', '')}?
+                        </DialogTitle>
+                        <DialogContent>
+                            This action will merge facility {toMergeName} with OAR
```

Comment:
This renders as a dense block of text. 

<img width="647" alt="Screen Shot 2019-06-19 at 2 27 22 PM" src="https://user-images.githubusercontent.com/17363/59802800-9abbd580-929e-11e9-82af-3d6ff2ec581f.png">


To help with this, consider removing the names of the facilities from the dialog content since they already appear in the title. Also consider splitting each of the three sentences into separate <p> elements.

<img width="616" alt="Screen Shot 2019-06-19 at 2 28 37 PM" src="https://user-images.githubusercontent.com/17363/59802806-9e4f5c80-929e-11e9-9d8e-526a8237b93b.png">

Reply:
I adjusted this in f9fd83a195fc9224020690d3f568c29224c2035a

![Screen Shot 2019-06-20 at 11 50 49 AM](https://user-images.githubusercontent.com/4165523/59862961-c9df4f00-9351-11e9-94d9-1c2c03e6e704.png)